sources/si.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "url": "http://www.arso.gov.si/xml/zrak/ones_zrak_urni_podatki_zadnji.xml",
+        "adapter": "slovenia",
+        "name": "Slovenian Environment Agency",
+        "city": "",
+        "country": "SI",
+        "description": "Official hourly air pollution data by the Slovenian Environment Agency",
+        "sourceURL": "http://www.arso.gov.si",
+        "resolution": "1 hr",
+        "contacts": [
+            "gp.arso@gov.si"
```

Comment:
`contacts` is currently unused, but it's purpose is to be used to notify a group of people if there is a problem with the source (generally those who wrote the adapter). This looks like it might be the official SI gov email address? Can you also add info@openaq.org to the list?